services/app-api/src/domain-models/healthPlanPackage.ts

```diff
@@ -74,6 +74,11 @@ function convertContractToUnlockedHealthPlanPackage(
 ): HealthPlanPackageType | Error {
     console.info('Attempting to convert contract to health plan package')
 
+    // Since drafts come in separate on the Contract type, we push it onto the revisions before converting below
+    if (contract.draftRevision) {
+        contract.revisions.unshift(contract.draftRevision)
```

Comment:
should we do anything else to verify that the first revision is indeed a draft here or can we be confident?